include/llama/RecordCoord.hpp

```diff
@@ -78,15 +96,15 @@ namespace llama
     template <typename L>
     using RecordCoordFromList = internal::mp_unwrap_sizes<L>;
 
-    /// Concatenate two \ref RecordCoord.
-    template <typename RecordCoord1, typename RecordCoord2>
-    using Cat = RecordCoordFromList<boost::mp11::mp_append<typename RecordCoord1::List, typename RecordCoord2::List>>;
+    /// Concatenate a set of \ref RecordCoord.
```

Comment:
You can use the plural form in Doxygen like this:

```suggestion
    /// Concatenate a set of \ref RecordCoord%s.
```

Reply:
That is great to know, I already wondered! I guess I should review the entire doxygen for this!